skimage/transform/tests/test_distance_transform.py

```diff
@@ -0,0 +1,28 @@
+from skimage.transform.distance_transform import (generalized_distance_transform,
+                                                  manhattan_dist,
+                                                  manhattan_meet)
+import numpy as np
+
+
+def test_1d():
+    case = [1, 0, 0, 0, 1, 1, 0, 1, 1, 0, 0, 0, 0, 1, 0, 1, 1, 0, 1, 0]
+    out_euc = [1.0, 0.0, 0.0, 0.0, 1.0, 1.0, 0.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 1.0, 1.0, 0.0, 1.0, 0.0]
+    out_man = [1.0, 0.0, 0.0, 0.0, 1.0, 1.0, 0.0, 1.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 1.0, 1.0, 0.0, 1.0, 0.0]
+    np.testing.assert_allclose(generalized_distance_transform(np.asarray(case)),np.asarray(out_euc))
+    np.testing.assert_allclose(generalized_distance_transform(np.asarray(case), dist_func=manhattan_dist, dist_meet=manhattan_meet),np.asarray(out_man))
+
+
+def test_2d():
+    case = [[1, 1, 1, 1, 1], [1, 1, 0, 0, 0], [1, 1, 1, 0, 0], [1, 0, 1, 0, 1], [0, 0, 1, 1, 1]]
+    out_euc = [[5.0, 2.0, 1.0, 1.0, 1.0], [4.0, 1.0, 0.0, 0.0, 0.0], [2.0, 1.0, 1.0, 0.0, 0.0], [1.0, 0.0, 1.0, 0.0, 1.0], [0.0, 0.0, 1.0, 1.0, 2.0]]
+    out_man = [[3.0, 2.0, 1.0, 1.0, 1.0], [2.0, 1.0, 0.0, 0.0, 0.0], [2.0, 1.0, 1.0, 0.0, 0.0], [1.0, 0.0, 1.0, 0.0, 1.0], [0.0, 0.0, 1.0, 1.0, 2.0]]
+    np.testing.assert_allclose(generalized_distance_transform(np.asarray(case)),np.asarray(out_euc))
+    np.testing.assert_allclose(generalized_distance_transform(np.asarray(case), dist_func=manhattan_dist, dist_meet=manhattan_meet),np.asarray(out_man))
+
+
+def test_3d():
+    case = [[[1, 0, 1], [1, 1, 1], [0, 0, 1]], [[1, 1, 0], [0, 1, 1], [0, 1, 1]], [[1, 1, 0], [0, 1, 1], [0, 1, 1]]]
+    out_euc = [[[1.0, 0.0, 1.0], [1.0, 1.0, 2.0], [0.0, 0.0, 1.0]], [[1.0, 1.0, 0.0], [0.0, 1.0, 1.0], [0.0, 1.0, 2.0]], [[1.0, 1.0, 0.0], [0.0, 1.0, 1.0], [0.0, 1.0, 4.0]]]
```

Comment:
maybe just use numpy to format these in a sensible way. It is really hard to read these as is.